src/app/page.tsx

```diff
@@ -70,8 +69,8 @@ export default function Home() {
   }, []);
 
   const handleSubmit = (value: number) => {
-    const battlePower = getBattlePower(value);
-    router.push(`/result?score=${battlePower}`);
+    sessionStorage.setItem('follower', String(value));
+    router.push('/scouter');
```

Comment:
フォロワー数のみなので発生確率は極めて低そうですが、セッションストレージは容量制限があるため、念の為エラーハンドリングしておくと良さそうです
```suggestion
    sessionStorage.setItem('follower', String(value));
    router.push('/scouter');
    try {
        sessionStorage.setItem('follower', String(value));
        router.push('/scouter');
    } catch (e) {
        console.error("セッションストレージへの保存に失敗しました:", e);
        setToastMessage("データの処理中にエラーが発生しました。もう一度お試しください。");
        setToastType("error");
    }
```

Reply:
セッションストレージに容量ってあるのか。この辺、調べよう